test/opt/instruction_test.cpp

```diff
@@ -577,4 +578,81 @@ TEST_F(OpaqueTypeTest, OpaqueStructTypes) {
     EXPECT_TRUE(type->IsOpaqueType());
   }
 }
+
+TEST_F(GetBaseTest, SampledImageLoad) {
```

Comment:
This is not a "load" but rather a sampling operation.  Please change the test name.

Reply:
In the `OpImageSampleImplicitLod` what does the "Lod" mean?  I understood that as a short form for "load".  I've changed the name, but I would still like this clarified.

Reply:
LOD = Level-of-detail;  a texture being sampled could have many slices at different resolutions, to be used when your view is "far" vs. "near".  See https://en.wikipedia.org/wiki/Mipmap